sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetOptions.scala

```diff
@@ -35,7 +35,7 @@ private[parquet] class ParquetOptions(
    * Compression codec to use. By default use the value specified in SQLConf.
    * Acceptable values are defined in [[shortParquetCompressionCodecNames]].
    */
-  val compressionCodec: String = {
+  val compressionCodecClassName: String = {
```

Comment:
@rxin This is a super minor but there are the same options to potentially rename in `OrcOptions`, `JsonOptions`, `CSVOptions` and `TextFileFormat` - [TextFileFormat.scala#L71](https://github.com/apache/spark/blob/3fbf5a58c236fc5d5fee39cb29e7f5c7e01c0ee7/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/text/TextFileFormat.scala#L71).

Also, I'd like to note, just in case, that the value in this is actually not the class name (it's something like `SNAPPY` and `LZO`) in case of `ParquetOptions` and `OrcOptions` whereas text-based ones are actual class names.